src/modules/twap/types.ts

```diff
@@ -22,3 +24,34 @@ export interface TWAPOrderStruct {
   t: number
   span: number
 }
+
+export enum TWAPOrderStatus {
+  WaitSigning = 'WaitSigning',
```

Comment:
What about: `WaitingSignature` ?

But also, in the Safe context, it could have all the signatures already and it's `WaitingExecution`, which in our case I assume it's the same thing.

Reply:
There are lots of other statuses, but I left them out of the scope of this task

Reply:
Will implement it in the next PR